### PR TITLE
[HOPSWORKS-1372] Upgrade the ELK stack to 7.2 OSS and use the OpenDis…

### PR DESCRIPTION
**Pull request by maismail**
…tro Security Plugin
https://logicalclocks.atlassian.net/browse/HOPSWORKS-1372
tests: https://github.com/logicalclocks/hops-testing/pull/565

### CODE REVIEW

**Comment by SirOibaf** [comment]
@kouzant can you have a look at this PR. Also can you make sure that everything works if we deploy in localhost mode?

File: files/default/kagent_utils/kagent_utils/kagent_config.py [hunk 1]
```diff
@@ -97,6 +97,9 @@ def read_conf(self):
             else:
                 self.host_id = self.hostname
 
+            self.elk_key_file = self._config.get('agent', 'elk-key-file')
+            self.elk_certificate_file = self._config.get('agent', 'elk-certificate-file')
+            self.elk_cn = self._config.get('agent', 'elk-cn')
```
Comment (kouzant) [inline files/default/kagent_utils/kagent_utils/kagent_config.py]: Can you also fix `test_kagent_config.py` and make sure it passes the tests?

File: files/default/csr.py [hunk 2]
```diff
@@ -328,4 +416,18 @@ def setup_logging(log_file, max_log_size, logLevel):
...
+        LOG.debug("ELK Admin certificate")
+        cert = ELKAdminCertificate(config)
+        
+        cert.create_csr()
+        
+        h = Host(config, cert, state_store)
+        with requests.Session() as session:
+            try:
+                h.generate_elkadmin_cert(session)
+                subprocess.call(config.keystore_script)
```
Comment (kouzant) [inline files/default/csr.py]: Why do you need to call the bash script here?

Comment (maismail) [reply files/default/csr.py]: Fixed

File: files/default/kagent_utils/kagent_utils/kagent_config.py (hunk 1 above)
Comment (maismail) [reply files/default/kagent_utils/kagent_utils/kagent_config.py]: Fixed.

**Pull request merged**